Tasks/AzureFunctionOnKubernetesV0/src/utils/commandHelper.ts

```diff
@@ -0,0 +1,112 @@
+import * as tl from "azure-pipelines-task-lib/task";
+import * as tr from "azure-pipelines-task-lib/toolrunner";
+import * as path from "path";
+import { DockerConnection } from "../dockerConnection";
+
+export class CommandHelper {
+    private funcPath: string;
+    public kubectlPath: string;
+
+    constructor() {
+        // Find the path of func and kubectl. This will throw if any of them is not present
+        // and hence the task will fail.
+        this.funcPath = tl.which('func', true);
+        this.kubectlPath = tl.which('kubectl', true);
+    }
+
+    public checkForErrors(execResults: tr.IExecSyncResult[], warnIfError?: boolean) {
```

Comment:
This logic is duplicate. You can move it to common and use it

Reply:
Done.